README.md

```diff
@@ -10,7 +10,7 @@
 
 Let's say you're building a course registration site for Flatiron. You need to build a list of available courses and include some details about each course.
 
-You need to build it so that when you click a course the details about the course disappear and reappear. Each course would need to have a click event that toggles the details. Flatiron also wants to be able to remove courses if they're not offered that semester, so there will need to be a button that removes that course (an `x`).
+You need to build it so that when you click a course, the details about the course disappear and reappear. Each course would need to have a click event that toggles the details. Flatiron also wants to be able to remove courses if they're not offered that semester, so there will need to be a button that removes that course (an `x`).
```

Comment:
It's grammatically sound as-is, and this change would create a comma splice. If you want, you could add a second comma:
>You need to build it so that, when you click a course, the details about the course...